docs/source/apps/supermarq/qcvv/qcvv_irb_css.ipynb

```diff
@@ -4,25 +4,27 @@
    "cell_type": "markdown",
```

Comment:
<p>trues -&gt; true: "We see that the measured two qubit gate error is in good agreement with the true value used by &nbsp;the simulator."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1068/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>